cpp/benchmarks/io/parquet/parquet_writer_chunks.cpp

```diff
@@ -26,6 +26,21 @@
 
 #include <nvbench/nvbench.cuh>
 
+std::string format_throughput(double bytes_per_second, int precision = 2)
+{
+  const std::vector<std::string> units = {"B/s", "KiB/s", "MiB/s", "GiB/s", "TiB/s"};
```

Comment:
Nit: May use stack allocation and [CTAD](https://en.cppreference.com/w/cpp/language/class_template_argument_deduction.html):
```suggestion
  std::array const units = {"B/s", "KiB/s", "MiB/s", "GiB/s", "TiB/s"};
```
Type of units will be deduced to `std::array<char const*, 2> const`